monkey/monkey_island/cc/services/edge/edge.py

```diff
@@ -0,0 +1,78 @@
+import copy
+from typing import Dict
+
+from bson import ObjectId
+from mongoengine import DoesNotExist
+
+from monkey_island.cc.models.edge import Edge
+
+RIGHT_ARROW = "\u2192"
+
+
+class EdgeService:
+
+    @staticmethod
+    def get_or_create_edge(src_node_id, dst_node_id, src_label, dst_label):
+        edge = False
```

Comment:
I think it'll be better to start with `None` and in the `finally` use `if edge is None`